apps/base/utils.py

```diff
@@ -18,20 +17,17 @@ def paginated_queryset(queryset, request):
     return (paginator, result_page)
 
 
-@deconstructible
```

Comment:
What will be the repercussions of this? Are you sure this wont impact the existing db or the app in any way?